Documentation/ceph-osd-mgmt.md

```diff
@@ -66,7 +66,7 @@ and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly c
 ### From the Toolbox
 
 1. Determine the OSD ID for the OSD to be removed. The osd pod may be in an error state such as `CrashLoopBackoff` or the `ceph` commands
-in the toolbox may show which OSD is `down`.
+in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you should run `kubectl -n <NS> scale deployment rook-ceph-deployment-<ID> --replicas=0` beforehand.
```

Comment:
```suggestion
in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you should run `kubectl -n rook-ceph scale deployment rook-ceph-osd-<ID> --replicas=0` beforehand.
```

The entire doc assumes rook-ceph so it's fine to keep it, also I think you meant `rook-ceph-osd`?

Reply:
Indeed, fixed.